### PR TITLE
Improve the journal log with grid

### LINKED ISSUE #5796: Improvements to Journal Log
Here is the current Journal Log for HEV:

image

There are a couple of things that make this a bit hard to use and improvements that I would like to see:

1. Let's make this a ui-grid.  It would make it easier to sort, filter, save views, count rows, etc, for free
     1. We should add in inline filtering on all columns at the same time.
     2. We should add sorting by clicking on the column headers for the date and transaction ID fields.
     3. In the grid footer, let's put the number of visible rows so that we know if we should increase our limit or not.
2. We should add a link (`bh-receipt`) to the receipt on the transaction ID.
3. It might also be useful to add an `actions` link that allows you to see the document in the journal, view the patient's record concerned, view the invoice, etc, like we do for the Journal.  These would link back to the correct registries or bring up the rendered PDF of the document.

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is related to #5796 

image

Close #5796

### CODE REVIEW

**Review by jniles — COMMENTED** [review]
All this looks good to me.

One request - could we add the `VO.TBA.123` identifier to the deleted records?  That will make it easier for users to see what was deleted.

**Comment by mbayopanda** [comment]
We have now information about the deleted record : 

image

File: client/src/modules/journal/journal_log.html [hunk 1]
```diff
@@ -27,41 +28,20 @@
...
-              </td>
-              <td>
-                <small translate>FORM.LABELS.BY</small>
-                <small>{{ row.display_name}}</small>
-              </td>
-            </tr>
-          </tbody>
-        </table>
-        
+        <div
```
Comment (jniles) [inline client/src/modules/journal/journal_log.html:31]: 👍

**Review by jniles — APPROVED** [review]
Looks good to me.  This is very useful 👍 


bors r+

**Pull request merged**